src/hooks/useGoTo.ts

```diff
@@ -0,0 +1,36 @@
+import { useNavigate } from 'react-router-dom';
+
+export function useGoToHome() {
+  const navigate = useNavigate();
+  return () => {
+    navigate('/');
+  };
+}
+
+export function useGoToProfile() {
+  const navigate = useNavigate();
+  return () => {
+    navigate('/my');
+  };
+}
+
+export function useGoToOrder() {
+  const navigate = useNavigate();
+  return (productId: number | string) => {
+    navigate(`/order/${productId}`);
+  };
+}
+
+export function useGoToTheme() {
+  const navigate = useNavigate();
+  return (themeId: number | string) => {
+    navigate(`/themes/${themeId}`);
+  };
+}
+
+export function useGoBack() {
+  const navigate = useNavigate();
+  return () => {
+    navigate(-1);
+  };
```

Comment:
```suggestion
export function useGoToHome() {
  const navigate = useNavigate();
  return () => {
    navigate('/');
  };
}

export function useGoToProfile() {
  const navigate = useNavigate();
  return () => {
    navigate('/my');
  };
}

export function useGoToOrder(productId: number | string) {
  const navigate = useNavigate();
  return () => {
    navigate(`/order/${productId}`);
  };
}

export function useGoToTheme(themeId: number | string) {
  const navigate = useNavigate();
  return () => {
    navigate(`/themes/${themeId}`);
  };
}

export function useGoBack() {
  const navigate = useNavigate();
  return () => {
    navigate(-1);
  };
```
이렇게 훅 자체에서 인자를 받아와도 된답니다!